### PR TITLE
Apparel rebalanced and various tweaks

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Additions

- Patched the ranged shield belts from VAE-Accessories that were somehow left out in the process, based on the patched vanilla shield belts

## Changes

- Rebalanced weight and worn bulk of flak pants, worn bulk of flak jackets and weight of composite and armor vests
- Adjusted the move speed of horses to make em more useful
- Removed the raid strategy tweaks made by CE, since 2 years ago there have been significant changes in raid strategy balance made in vanilla game which our patches screw with that.

## Reasoning

- Balance reasons.

## Alternatives

- None.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors

### CODE REVIEW

File: Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml [hunk 1]
```diff
@@ -112,6 +112,30 @@
 				</value>
 			</li>
 
+			<!-- == Ranged Shield Pack == -->
+
+			<Operation Class="PatchOperationAdd">
```
Comment (N7Huntsman) [inline Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml:117]: These should be `<li>` tags, not operations.

File: Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml [hunk 2]
```diff
@@ -112,6 +112,30 @@
...
+				<xpath>Defs/ThingDef[defName="VAEA_Apparel_RangedShieldBelt"]/statBases/EnergyShieldRechargeRate</xpath>
+				<value>
+					<EnergyShieldRechargeRate>0.195</EnergyShieldRechargeRate>
+				</value>
+			</Operation>
+
+			<Operation Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="VAEA_Apparel_RangedShieldBelt"]/statBases/EnergyShieldEnergyMax</xpath>
+				<value>
+					<EnergyShieldEnergyMax>1.35</EnergyShieldEnergyMax>
```
Comment (N7Huntsman) [inline Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml:135]: What's the reason for this change, and what's the impact on balance in combat?

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Mass changes to apparel seem fine. Some stuff needs fixed.

What're the actual changes in gameplay for removing the tweaks to raid stuff?

File: Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml (hunk 2 above)
Comment (ItsSamGray) [reply Patches/Vanilla Apparel Expanded - Accessories/Apparel_Utility.xml:135]: Ranged shield belts were left unpatched and with the patched values of vanilla shield belts, the gap between them got much wider and ranged ones remained quite weak.

**Review by N7Huntsman — APPROVED** [review]
